partiql-tests-data/eval/query/select/select.ion

```diff
@@ -955,7 +955,7 @@ select_join::[
 select_bugs::[
   {
     name:"MYSQL_SELECT_BUG_01",
-    statement:"select * from t1 left join t2 on t1.a = t2.c where t2.d in (4);",
```

Comment:
Semicolon in statement not part of SQL or PartiQL spec. These conformance tests should not test for that syntax